ai-machine-learning/thoth-thamos/setup.md

```diff
@@ -0,0 +1,28 @@
+Before start using the Thamos let's setup the environment.
```

Comment:
change to: "Before we start using Thamos, let's give a couple of minutes for the environment to setup."?

Reply:
Sounds good!